apps/web/src/pages/ApplicantDashboardPage/components/PoolCandidateSearchRequestPreviewListItem.tsx

```diff
@@ -0,0 +1,157 @@
+import { IntlShape, useIntl } from "react-intl";
+import { useState } from "react";
+
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  PreviewListItemSearchRequestFragment,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { PreviewList } from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import { formatClassificationString } from "~/utils/poolUtils";
+
+import ReviewTalentRequestDialog from "./ReviewTalentRequestDialog";
+import { deriveChipSettings } from "./utils";
+
+const PreviewListItemPoolCandidateSearchRequest_Fragment = graphql(
+  /* GraphQL */ `
+    fragment PreviewListItemSearchRequest on PoolCandidateSearchRequest {
+      id
+      jobTitle
+      status {
+        value
+      }
+      requestedDate
+      applicantFilter {
+        qualifiedClassifications {
+          group
+          level
+        }
+      }
+      initialResultCount
+    }
+  `,
+);
+
+function buildTitle(
+  request: PreviewListItemSearchRequestFragment,
+  intl: IntlShape,
+): string {
+  const classifications = unpackMaybes(
+    request.applicantFilter?.qualifiedClassifications,
+  );
+  const classificationString =
+    classifications.length == 1
+      ? formatClassificationString(classifications[0])
+      : null;
+
+  const jobTitleString =
+    request.jobTitle ?? intl.formatMessage(commonMessages.notProvided);
+
+  const completedTitle =
+    typeof classificationString === "string"
+      ? `${classificationString}${intl.formatMessage(commonMessages.dividingColon)}${jobTitleString}`
+      : jobTitleString;
+
+  return completedTitle;
+}
+
+interface PoolCandidateSearchRequestPreviewListItemProps {
+  poolCandidateSearchRequestQuery: FragmentType<
+    typeof PreviewListItemPoolCandidateSearchRequest_Fragment
+  >;
+}
+
+const PoolCandidateSearchRequestPreviewListItem = ({
+  poolCandidateSearchRequestQuery,
+}: PoolCandidateSearchRequestPreviewListItemProps) => {
+  const intl = useIntl();
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
```

Comment:
Is doing it this way going to lose focus?

Reply:
Yep, this causes focus lock.  I think you'll need to get rid of the useState here and pass a unique trigger button into the dialog.

Reply:
Nice catch Peter :+1: Fixed in [Fix dialog trigger focus](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14135/commits/fa6a1c385988c0f168dfb58db7936330f2f05dcc)